Setup Page Structure Using Grid Template

devmailar opened the pull request.

devmailar committed:
Create .gitignore

devmailar committed:
Create grid layout structure for the website

This commit introduces the `index.html` file, which establishes the basic structure of the Group2 webpage, including a navigation bar, header, main content area, and footer.

Additionally, the `index.css` file is added to define styles for the body and grid layout, ensuring a visually appealing presentation with distinct areas for navigation, header, main content, and footer.

devmailar commented:
LGTM!

Git-It-Got-It-Good commented:
I think the grid layout so far looks like SE's template (excellent), and the colors on display so far match SA's list (also excellent). Not sure if we will add an aside to become our "What is Copyright?..." segment on the home page, or if that will just be a part of our main -- that idea was still in progress.

I agree with ST's comment that I would like to expand the max width; maybe to at least double what it currently is.

devmailar commented:
> I think the grid layout so far looks like SE's template (excellent), and the colors on display so far match SA's list (also excellent). Not sure if we will add an aside to become our "What is Copyright?..." segment on the home page, or if that will just be a part of our main -- that idea was still in progress.
> 
> I agree with ST's comment that I would like to expand the max width; maybe to at least double what it currently is.

You mean our website would use full width of the user viewport / screen size?

Git-It-Got-It-Good commented:
> You mean our website would use full width of the user viewport / screen size?

That's an option (at least for the desktop  view), or we could try setting the max-width at 80em or 85 em and see what it looks like

Git-It-Got-It-Good committed:
Add files via upload

Added files for second rough draft of page 1; added an aside to the grid; added 2 images to help gauge placement and sizing

devmailar commented:
> > You mean our website would use full width of the user viewport / screen size?
> 
> That's an option (at least for the desktop view), or we could try setting the max-width at 80em or 85 em and see what it looks like

I saw your [PR ](https://github.com/iui-hh-2025/group2/pull/3) what if we merge that to here so this issue would also be solved

Git-It-Got-It-Good committed:
Merge branch '1-setup-page-structure-using-grid-template' into page-1-draft-2

sankenbauer reviewed (commented):
Looks good to me!

Git-It-Got-It-Good committed:
Merge pull request #3 from iui-hh-2025/page-1-draft-2

Page 1, Draft 2
Merging to 1-setup-page-structure-using-grid-template
